phpBB/includes/acp/acp_profile.php

```diff
@@ -757,6 +757,10 @@ function main($id, $mode)
 				$s_one_need_edit = true;
 			}
 
+			if (!isset($this->type_collection[$row['field_type']]))
+			{
+				continue;
+			}
```

Comment:
This code has already been merged by https://github.com/phpbb/phpbb/pull/4555
No need to re-include it here, or it will create merge conflicts or duplicate code.
Leftover from the past... the original commit was too old, and forgot to remove this code until today.